modules/oe_whitelabel_list_pages/tests/src/Functional/ListPagesTest.php

```diff
@@ -72,36 +109,81 @@ public function testListPage(): void {
     $list_page->get('emr_entity_metas')->attach($list_page_entity_meta);
     $list_page->save();
 
-    $this->drupalGet('node/' . $list_page->id());
+    return $list_page;
+  }
 
-    // Assert the left column.
-    $assert_session->elementExists('css', 'div.row > div.col-12.col-lg-3');
-    // Assert offcanvas.
-    $assert_session->elementExists('css', 'div.bcl-offcanvas');
-    $assert_session->elementTextEquals('css', 'h4.offcanvas-title', 'Filter options');
-    $assert_session->elementExists('css', 'button.btn-light > svg');
-    $assert_session->elementTextEquals('css', 'button[data-bs-toggle="offcanvas"]', 'Filters');
-    // Assert Filters.
-    $assert_session->elementExists('css', 'input[name="oe_sc_news_title"]');
-    $assert_session->elementExists('css', 'input[data-drupal-selector="edit-submit"]');
-    $assert_session->elementExists('css', 'input[data-drupal-selector="edit-reset"]');
+  /**
+   * Asserts the title in the facets summary.
+   *
+   * @param int $expected_count
+   *   Expected number of results to be reported in the title.
+   * @param \Behat\Mink\Element\ElementInterface $container
+   *   Document to check against.
+   */
```

Comment:
Description for `$container` parameter is not correct.
The element passed here is not the "document", it is the "right column", so `'div.row > .col-lg-9'`.
I wonder if we even need this parameter or if each assert method should do its own `$this->assertSession()->elementExists('div.row > .col-lg-9')->...`, or use a helper to get that container element.
A parameter is only useful if we could also pass something else instead.
But I'm ok to keep it like this if we want to move forward.

Reply:
It is agnostic of the right column, that was the point behind making it a parameter.
I don't mind what we did with the "getSearchTopRegion" but here I want to put emphasis on the structure of the page:
get left column
  assert stuff using that as context
get right column
  assert stuff using that context
  
so either I use it as a parameter to point out that this stuff is there, in the $right_column or I add it in the helper names, like assertRightColumnFacetsSummaryTitle and inside do getRightColumn(), up 2 u Mr Reviewer